h/templates/activity/search.html.jinja2

```diff
@@ -273,38 +273,25 @@
 
     {{ tags_section() }}
 
-    {# if the group has a concept of members display members and join text #}
-    {% if group.members %} 
-      {{ group_users("Members", group.members, group.creator) }}
-      <section class="search-result-sidebar__section">
-        <h3 class="search-result-sidebar__subtitle">
-          {% trans %}Invite new members{% endtrans %}
-        </h3>
-        <p class="search-result-sidebar__share-link">
-          {% trans %}Sharing the link lets people join this group:{% endtrans %}
-        </p>
-    {# if the group has no concept of members display moderators and share text #}
-    {% else %}
-      {{ group_users("Moderators", group.moderators, group.creator) }}
-      <section class="search-result-sidebar__section">
-        <h3 class="search-result-sidebar__subtitle">
-          {% trans %}Share group{% endtrans %}
-        </h3>
-        <p class="search-result-sidebar__share-link">
-          {% trans %}Sharing the link lets people view this group:{% endtrans %}
-        </p>
-    {% endif %}
-    <div class="group-invite__container js-copy-button">
-      <input class="group-invite__input js-select-onfocus"
-             data-ref="input"
-             value="{{ group.url }}">
-      <button class="group-invite__clipboard-button"
-              data-ref="button"
-              title="Copy to clipboard">
-        {{ svg_icon('copy_to_clipboard', 'group-invite__clipboard-image') }}
-      </button>
-    </div>
-  </section>
+    {{ group_users(group_users_display_info.title, group_users_display_info.members, group.creator) }}
+    <section class="search-result-sidebar__section">
+      <h3 class="search-result-sidebar__subtitle">
+        {{ group.share_subtitle }}
+      </h3>
+      <p class="search-result-sidebar__share-link">
+        {{ group.share_msg }}
+      </p>
+      <div class="group-invite__container js-copy-button">
+        <input class="group-invite__input js-select-onfocus"
+               data-ref="input"
+               value="{{ group.url }}">
+        <button class="group-invite__clipboard-button"
+                data-ref="button"
+                title="Copy to clipboard">
+          {{ svg_icon('copy_to_clipboard', 'group-invite__clipboard-image') }}
+        </button>
+      </div>
+    </section>
```

Comment:
Good to see that this bit of template code got much shorter. I also think it was probably a good idea to move `share_subtitle` and `share_msg` into Python where they can be tested :+1: